### ISSUE TITLE
Waterways now Warn about connectivity and completness 

### ISSUE THREAD

**Issue by jjiglesiasg** [facebook/Rapid#1018]
### Description

Now there is a Warning for waterways that are "unreacheable" and "has no outlet" even been intermittent the waterway.

This to me, seems to be Unnecessary. Mostly of the waterways in the map are NOT completed at all; and to have warnings for dozen or even hundreds of waterways is just basically annoying that serve no practical porpoises. For each ruteable waterway in the world there are even millions of non ruteable ones, ilogic to warn for each and every one.

These warnings should arise or show up ONLY when the waterway has conditions like boat=yes or some kind of "navegability" tagging, otherwise is just NOISE.

### Version

2.0.3

### What browser are you seeing the problem on? What version are you running?

Version 114.0.5735.199 (Official Build) (64-bit)

### Steps to reproduce

I draw some portions of some waterways (Intermitent) and somehow now there is Warnings. at the moment of saving and uploading

### The browser URL at the time you encountered the bug

https://rapideditor.org/rapid#background=Bing&datasets=fbRoads&disable_features=boundaries&map=19.00/-17.64996/-67.99577
![Screenshot (46)](https://github.com/facebook/Rapid/assets/22849578/d89637d8-9527-4e83-92c4-20fc667eab37)

**Comment by bhousel** [comment]
Makes sense to me!  I added some code to ignore `intermittent=yes` from the crossing_ways and disconnected_ways checks.

**Comment by jjiglesiasg** [comment]
This could be closed. Unreachable is just for ways (roads) and not for waterways. The Warning was incorrect because the element has both tags uncorrectly. That's why the warning triggers
